room.py

```diff
@@ -2,12 +2,13 @@
 import string
 
 from user import User
-
+	
 
 class Room:
     """
     Class for representing "Rooms"
     """
+	__rooms = {}
```

Comment:
It looks like you've got mixed spaces & tabs for indents in this file

Reply:
Darn it notepad++